software/22.Serverless/SETUP.md

```diff
@@ -0,0 +1,57 @@
+# Workshop 22 - SETUP
+
+## Dependencies
+Please make sure you have the following programs installed:
+- [node (version 14 or higher)](https://github.com/nodejs/node): javascript interpreter
+- [npm](https://www.npmjs.com/): node package manager
+- [yarn](https://yarnpkg.com/): a better package manager
+
+To install `node` and `npm`:
+- under fedora: `sudo dnf install nodejs`.
+- under ubuntu: `sudo apt install nodejs npm`.
+
+To install `yarn`: `npm install --global yarn`
+
+## Github
+> You will need a github repository in order to use Netlify 📂
+- Create a Github repositoty (named: `Serverless-Gifs-App` for example) and clone it with `git clone [YOUR REPO SSH KEY]`
+- Download the [files](./Netlify-App.zip)
+- Run `mv Netlify-App/* Netlify-App/.* [YOUR REPO NAME]`
+- Run `cd [YOUR REPO NAME]`
+- Run `yarn install` to install the dependencies
+> Now we gonna verify that the app is working 👍
+- Run `yarn start` to start the server
```

Comment:
Explain the command using the following structure:  https://github.com/PoCInnovation/Workshops/blob/master/software/18.Kafka/SETUP.md#install-the-go-programming-language

You should do a code snippet with space and comment to explain what they do (and why if necessary)